### ISSUE TITLE
Weapons too accurate when in motion

### ISSUE THREAD

**Issue by chrisnew** [eukara/FreeCS#89]
When in motion like running or in air, weapons are way too accurate.

Best example is AWP in air. It’s still too precise even without using the scope.

**Comment by eukara** [comment]
Yeah, right now there is no differentiation between standing/running/walking. If anyone has any solid multipliers... let me know! I'll create some placeholder values for now.
